.vscode/settings.json

```diff
@@ -48,5 +48,6 @@
 	"cSpell.words": [
 		"Cupertino",
 		"switchtile"
-	]
+	],
+	"dart.flutterSdkPath": "/home/dmitry.detukov/opt/flutter/3.3.6-x64"
```

Comment:
Is this needed?

Reply:
Maybe not, I checked the program works without it:
"cSpell.words": [
		"Cupertino",
		"switchtile"
	]

Reply:
I mean `/home/dmitry.detukov/opt/flutter/3.3.6-x64`

Reply:
No, i will fix it now

Reply:
Looks good )